roles/setup-site/tasks/main.yml

```diff
@@ -77,7 +78,7 @@
   retries: "{{ wait_time }}"
   delay: 50
   register: bootstrap_status
-  until: bootstrap_status.stdout | search('Successful')
+  until: bootstrap_status.stdout is search('Successful')
```

Comment:
Is this a ansible version change thing? Why the change in the pipe?

Reply:
I was getting deprecated warnings

Reply:
Good to know. Thank you.